examples/ppo/ppo_lib.py

```diff
@@ -0,0 +1,251 @@
+"""Library file which executes the PPO training"""
+
+import functools
+from typing import Tuple, List
+import jax
+import jax.random
+import jax.numpy as jnp
+import numpy as onp
+import flax
+from flax import nn
+from flax.metrics import tensorboard
+from flax.training import checkpoints
+import ml_collections
+
+import agent
+import test_episodes
+
+@functools.partial(jax.vmap, in_axes=(1, 1, 1, None, None), out_axes=1)
```

Comment:
Nit: I think having the jit on the outside makes more sense. I think Jax is smart about this but compile(vectorized code) makes more sense than vectorize(compiled_code) IMO

Reply:
Changed in https://github.com/google/flax/pull/470/commits/d444075ee4337ceb2cc21ce071622ba0ed62d745